src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,357 @@
+---
+tags:
+- libp2p
+- Ethereum
+- Merge
+title: libp2p & Ethereum (the Merge)
+description:
+date: 2023-01-01
+permalink: ""
+translationKey: ''
+header_image: /libp2p_Ethereum_header.png
+author: Prithvi Shahi
+---
+
+If you've kept up with developments in Web3, you've likely heard of a significant event that occurred a few months ago: the [Paris upgrade](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md), more widely known as [The Merge](https://ethereum.org/en/upgrades/merge/), on the [Ethereum network](https://ethereum.org/en/).
+This was a historic event for Ethereum and one that was years in the making ([Google even had a countdown](https://decrypt.co/109428/google-counts-down-to-ethereum-merge-with-search-engine-easter-egg) for it.)
```

Comment:
Just to make sure this is not missed before merging, note that the new-line-per-sentence is rendered as an actual new-line by the blog software, contrary to GitHub Markdown which merges the two sentences into a single paragraph.

Reply:
Completely agreed.  This was my first observation when looking at the preview: https://bafybeihd4hahred6rxx2kprkzkgjml7hdtl6bcxgbyw3vjrwtqzfwazjhq.on.fleek.co/libp2p-and-ethereum/

I think as long as you dont' have any spaces at the end of your lines, it will render correctly.

Reply:
Right I'm aware of this, will be addressing it at the end. Some multilines have already been collapsed into paragraphs